lms/static/sass/base/_variables.scss

```diff
@@ -448,6 +448,6 @@ $red1: #D0021B;
 
 // +feature: student notes
 // --------------------
-$student-notes-highlight-color-base: saturate($yellow, 50%);
-$student-notes-highlight-color: tint($student-notes-highlight-color-base, 60%);
+$student-notes-highlight-color-base: saturate($yellow, 65%);
```

Comment:
NOTE: getting all designer-levels of picky with the notes yellow highlight color :P